services/cypress/integration/stateWorkflow/stateSubmissionForm/accessiblility.spec.ts

```diff
@@ -1,73 +1,114 @@
+import { stateUser } from '../../../utils/apollo-test-utils';
+
 describe('state user in state submission form', () => {
     beforeEach(() => {
         cy.stubFeatureFlags()
         cy.interceptGraphQL()
     })
-    // SKIPPED because we currently have a bug that looks related to https://github.com/mfrachet/cypress-audit/issues/136#issuecomment-1311236777
-    // this started when we upgraded node and serverless and started default to ipv6 which prevented tests
-    // current this keeps pa11y from running
-    it.skip('can fill out contract only submission with no a11y errors', () => {
-        // goal of this test is to check every single form page and navigation (going backwards, forwards or save as draft with new info)
-
+    it('has no a11y violations on submission form with form input errors', () => {
+        // 438-attestation still needs to go through design, there is an a11y violation for links and spacing
+        cy.interceptFeatureFlags({'438-attestation': false})
         cy.logInAsStateUser()
 
-        // Start a base contract only submissions
-        cy.startNewContractOnlySubmissionWithBaseContract()
-
+        // Inject the axe run-time
+        cy.injectAxe()
 
-        cy.findByRole('heading', {
-            level: 2,
-            name: /Contract details/,
-            timeout: 10_000,
-        })
+        // Start a base contract only submissions
+        cy.findByTestId('state-dashboard-page').should('exist')
+        cy.findByRole('link', { name: 'Start new submission' }).click()
 
-            // check contract details a11y
-            cy.pa11y({
-                hideElements: '.usa-step-indicator',
-            actions: ['wait for element #form-guidance to be visible'],
-                threshold: 6,
-            hideElements: '.usa-step-indicator',
-        })
-        cy.deprecatedNavigateV1Form('BACK')
-          // check submissionType a11y
-          cy.pa11y({
-            hideElements: '.usa-step-indicator',
-        actions: ['wait for element #form-guidance to be visible'],
-            threshold: 6,
-        hideElements: '.usa-step-indicator',
-    })
+        // Check accessibility on Submission type page
+        cy.findByRole('heading', { level: 1, name: /New submission/ })
+        cy.findByRole('button', {
+            name: 'Continue',
+        }).should('not.have.attr', 'aria-disabled')
+        cy.findByRole('button', {
+            name: 'Continue',
+        }).safeClick()
+        cy.checkA11yWithWcag22aa()
 
-            cy.deprecatedNavigateV1Form('CONTINUE')
-            cy.deprecatedNavigateV1Form('CONTINUE')
+        cy.fillOutContractActionAndRateCertification()
+        cy.deprecatedNavigateV1Form('CONTINUE_FROM_START_NEW')
```

Comment:
I thought Pearl merged her work and start, submission and contract details are are all on v2. I think this could be using the v2 helpers?

Reply:
I don't see a v2 of `fillOutContractActionAndRateCertification`. I do see `startNewContractOnlySubmissionWithBaseContractV2`, but the purpose of the code here is to:
- check a11y violations on the empty form with field errors
- fill out the form and continue so that we have a submission to go to each form page.

Using `startNewContractOnlySubmissionWithBaseContractV2` would fill this out and continue before we could check for a11y violations.

Reply:
Look at `navigateCOntractAndRatesForm` and `navigateContractForm`.